ui/app/routes/variables/variable.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+import withForbiddenState from 'nomad-ui/mixins/with-forbidden-state';
+import WithModelErrorHandling from 'nomad-ui/mixins/with-model-error-handling';
+
+export default class VariablesVariableRoute extends Route.extend(
+  withForbiddenState,
+  WithModelErrorHandling
+) {
+  model(params) {
+    return this.store.findRecord('variable', params.path, { reload: true });
```

Comment:
Are we missing injecting the `store` service here? Do we need to declare that this is an `async` model?

https://guides.emberjs.com/release/routing/specifying-a-routes-model/